static/css/stac-ui.css

```diff
@@ -0,0 +1,1796 @@
+    place-content: center;
+    width:12px;
+    height: 12px;
+    border-radius: 50px;
+    border: solid;
+    border-width: 2px;
+    border-color: var(--primary);
+}
+
+.radio-button-small::before {
+    content: "";
+    width: 6px;
+    height: 6px;
+    border-radius: 50%;
+    transform: scale(0);
+}
+
+.radio-button-small:checked::before {
+    transform: scale(1);
+    background-color: var(--primary);
+}
+
+.margin-radio-left{
+    margin-right: 7px;
+}
+
+.margin-radio-right{
+    margin-left: 7px;
+}
+/*End Radio button*/
+
+
+/*Input date*/
+.datepicker-container{
+    display: flex;
+    flex-direction: column;
+}
+
+.input-date-reg{
+    width: 123px;
+    height: 46px;
+    padding: 12px 14px;
+    color: var(--gray-500);
+    background-color: var(--white);
+    border-radius: 9px;
+    border: solid;
+    border-width: 1px;
+    border-color: var(--gray-400);
+    text-align: center;
+}
+
+.timepicker-container{
+    display: flex;
+    flex-direction: column;
+}
+
+.input-time-sm{
+    width: 100px;
+    height: 46px;
+    padding: 12px 14px;
+    color: var(--gray-500);
+    background-color: var(--white);
+    border-radius: 9px;
+    border: solid;
+    border-width: 1px;
+    border-color: var(--gray-400);
+    text-align: center;
+}
+
+/*Result Display*/
+/*Display as row*/
+.result-display-row{
+    display: flex;
+    flex-direction: row;
+}
+
+.result-display-row div:not(:last-child){
+    margin-right: 20px;
+}
+
+.result-link-container{
+    display: flex;
+    flex-direction: row;
+}
+
+.result-link{
+    color: var(--primary);
+    text-decoration: none;
+}
+
+.result-link-title{
+    width: 245px;
+    height: 42px;
+    color: var(--primary);
+    text-decoration: none;
+    text-align: center;
+}
+
+.result-format{
+    display: flex;
+    flex-direction: row;
+}
+
+.result-format-row{
+    display: flex;
+    flex-direction: column;
+}
+
+.result-text{
+    color: var(--gray-700);
+}
+
+.icon-more-information{
+    color: var(--primary);
+}
+
+.margin-icon-more-information{
+    margin-left: 15px;
+}
+
+/*Display as card*/
+.result-display-card{
+    width: 453px;
+    height: 167px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border-radius: 4px;
+    border: solid ;
+    border-width: 1px;
+    border-color: var(--display-card-border);
+}
+
+.display-card-time-range{
+    display: flex;
+    flex-direction: row;
+    margin-top: 14px;
+    margin-bottom: 14px;
+}
+
+.display-card-time-range div:not(:last-child){
+    margin-right: 5px;
+}
+
+
+/*Date Picker*/
+/*Override Flatpickr CSS*/
+.flatpickr-day{
+    border: solid;
+    border-width: 1px;
+    border-color: var(--background-transparent);
+    border-radius: 9px;
+}
+
+.flatpickr-day:hover{
+    background-color: var(--indigo250-opacity-light);
+    border: solid;
+    border-width: 1px;
+    border-color: var(--indigo250-opacity-light);
+    border-radius: 9px;
+}
+/*
```

Comment:
remove commented code

Reply:
Sorry, this is for adjusting the arrow buttons in the calendar month menu.  I was having trouble getting the spacing the way Shruti has in the design.

Added a comment for this.   

Hopefully the default position fits with Shruti's design and this part can be deleted.